docs/source/commandline/commandline-all.rst

```diff
@@ -47,57 +57,84 @@ Configuration commands
 
 
 
-Frontend commands
+Frontend Commands
 -----------------
 
 Note: By default, all the '**-list**' commands below write info to the screen. If you wish to write the info to a textfile instead, add this to the end of your command:
 
-  > filename
+  **> filename**
+
+...where 'filename' is the textfile's path and name (e.g., list.txt).
 
-...where 'filename' is the textfile's path and name (e.g., c:\\mame\\list.txt).
+Example:
+
+|  **mame -listcrc puckman > list.txt**
+|
+| This creates (or overwrites if it's already there) list.txt and fills the file with the results of **-listcrc puckman**. In other words, the list of each ROM used in Puckman and the CRC for that ROM are written into that file.
+|
 
+.. _mame-commandline-listxml:
```

Comment:
This description is kind of dated.  It lists information about systems and devices (ixnay on the "game" terminology).  If you specify a system name or glob pattern it will only list matching systems and the devices they use.

This command can accept multiple system names/glob patterns, so remember the ellipsis.

Actually, the whole lot needs the game name terminology fixed.  MAME emulates systems, not games.

Reply:
Working on this still-- I just need to do a final pass to make sure I got all the stuff from all three commandline lists.